ClosedXML_Tests/Excel/Cells/XLCellTests.cs

```diff
@@ -1083,5 +1083,24 @@ public void TryGetValueFormulaEvaluation()
                 Assert.IsTrue(A3.TryGetValue(out String _));
             }
         }
+
+        [Test]
+        public void CopyFromInternalNullTest()
```

Comment:
I'm still curious as to what usage of the public API led to the exposure of the bug. The use of internals in the test project isn't disallowed, but ideally we want a test using only the public API. Maybe we should discuss in the main thread first. If you can show me the code sample to reproduce the issue, that would be great.

Reply:
Yesterday I have linked [PR](https://github.com/ClosedXML/ClosedXML.Report/pull/169) and also @b0bi79 has linked [an issue](https://github.com/ClosedXML/ClosedXML.Report/issues/170) in ClosedXML.Report.
Maybe throwing exception on null `otherCell` is better, but I am not too familiar with other usage scenarios.

Reply:
NRE or other unhandled exceptions are no good for the internals nor the public API.
I think of this PR as a declaration of behavior: do nothing or throw ArgumentNullException.
Of course, the decision is up to you.